src/WebJobs.Script.WebHost/Configuration/ResponseCompressionOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Hosting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    internal sealed class ResponseCompressionOptions : IOptionsFormatter
```

Comment:
curious if there is a specific reason for making it a `sealed` class?

Reply:
Sealed classes provide a slight performance benefit.  Any new classes we write should be sealed unless there is a need for a derived class. a sealed class can be unsealed when/if needed in the future.